odl/solvers/iterative/iterative.py

```diff
@@ -209,17 +208,15 @@ def conjugate_gradient(op, x, rhs, niter=1, callback=None):
 
 
 def conjugate_gradient_normal(op, x, rhs, niter=1, callback=None):
-    """Optimized implementation of CG for the normal equation.
-
-    This method solves the normal equation
+    """Optimized implementation of CG for the normal equations.
 
-    :math:`A^* A x = A^* y`
+    This method solves the inverse problem (of the first kind)
 
-    to the inverse problem (of the first kind)
+    ``A(x) == rhs``
```

Comment:
use `::` (with prepended space) on the previous line, and no \`\`. That will typeset this line as code without printing a colon.